### DISCUSSION TITLE
Brightness

### DISCUSSION THREAD

**Discussion by rygor-thomas** [SpiralLinux/SpiralLinux-project#503] [Q&A] [answered]
Does anyone know how to fix the screen brightness ? Every time I log in to my Pc the brightness has gone back to its original setting (about 50%) which is way too bright for my old eyes!
I am using the LXQT version of Spiral Linux on a iMac of 2013 vintage with an i5 processor and 32Gb RAM.
I must congratulate the developer on a very nice distro all round.

**Comment by marwenius** [comment, marked as answer]
I found a solution on Reddit where one person also had a Macbook:
https://www.reddit.com/r/linuxquestions/comments/b78sta/comment/ejqd1p2/

In the terminal:
```
sudo nano /etc/default/grub
```
Find the line GRUB_CMDLINE_LINUX_DEFAULT and add the following within the quotation marks:
```
acpi_backlight=native
```
This command must be executed after saving and exiting (CTRL + S, CTRL + X):
```
sudo update-grub
```
Then restart (twice if necessary) to test whether the brightness settings are now applied.

This article also deals with this issue:
https://www.reddit.com/r/pop_os/comments/11ok9rs/amdgpu_backlight_failing_to_initialize_macbook/

**Reply by geckolinux** [reply]
Thanks a lot @marwenius for the helpful response.
Also just a note that SpiralLinux has **Grub Customizer** pre-installed that allows for easily changing the default kernel parameters.

**Reply by rygor-thomas** [reply]
Many thanks to you Marwenius. That solved my problem, only not in the way I was expecting, the brightness slider is still at 50% but the backlight slider is now on 0%. At least now I am not dazzled!

**Comment by rygor-thomas** [comment]
Thanks for your suggestion Marwenius. I will look into this later on and post back.
